app/src/main/java/com/nemo/androiduitraining/view/util/ViewExt.kt

```diff
@@ -0,0 +1,25 @@
+package com.nemo.androiduitraining.view.util
+
+import android.view.View
+import androidx.databinding.BindingAdapter
+import java.time.Instant
+
+/**
+ * 連打対策付きクリックイベント
+ */
+@BindingAdapter("onClick")
+fun View.setOnClickListenerWithoutDoubleTap(onClickListener: View.OnClickListener?) {
+    setOnClickListener(DoubleTapOnClickListener(onClickListener))
+}
+
+class DoubleTapOnClickListener(val listener: View.OnClickListener?) : View.OnClickListener {
+    private var lastTapAt = 0L // ms
+
+    override fun onClick(p0: View?) {
```

Comment:
[質問]
`p0` の変数名は変更できないでしょうか？
もしできるなら `view` などのほうがわかりやすいです。

```suggestion
    override fun onClick(view: View?) {
```

Reply:
できます！
直します！